enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -576,7 +569,10 @@ func (p *PebbleCache) deleteOrphanedFiles(quitChan chan struct{}) error {
 
 			dupPartitionPattern := fmt.Sprintf("%s/%s", part.ID, part.ID)
 			pathHasDupPartitionID := strings.Contains(path, dupPartitionPattern)
```

Comment:
We can remove this - the migrated caches shouldn't have the duplicated partitions